src/platform/packages/shared/kbn-monaco/src/esql/lib/esql_theme.ts

```diff
@@ -96,12 +93,12 @@ export const buildESQLTheme = ({
           'dev_rrf',
           'dev_completion',
         ],
-        euiThemeVars.euiColorAccentText,
+        euiTheme.colors.accent,
```

Comment:
```suggestion
        euiTheme.colors.textAccent,
```

Reply:
This will fix the pinkish

<img width="754" alt="image" src="https://github.com/user-attachments/assets/adb797e3-e896-4f55-9175-31f0a666b063" />